docs/examples/warming-stripe.json

```diff
@@ -0,0 +1,264 @@
+{
+  "$schema": "https://vega.github.io/schema/vega/v5.json",
+  "description": "Warming Stripes Chart. Originally designed by Ed Hawkins.",
+  "usermeta": {
+    "Author": "Andrzej Leszkiewicz",
+    "Github": "https://github.com/avatorl/Deneb-Vega/",
+    "License": "CC BY 4.0 (Share and Adapt with Attribution)",
+    "Original Idea Author": "Ed Hawkins",
+    "Original Idea URL": "https://www.climate-lab-book.ac.uk/2018/warming-stripes/",
+    "Read More URL": "https://en.wikipedia.org/wiki/Warming_stripes",
+    "Data Source URL": "https://climate.nasa.gov/vital-signs/global-temperature/"
+  },
+  "background": "white",
+  "width": 650,
+  "height": 200,
+  "padding": 10,
+  "autosize": "pad",
+  "config": {},
+  "signals": [
+    {"name": "fontMain", "value": "Tahoma"},
+    {"name": "labelColor", "value": "#808080"},
+    {"name": "labelDataSrcColor", "value": "#CCCCCC"},
+    {"name": "basePeriodColor", "value": "black"},
+    {"name": "titleFontSize", "value": "16"},
+    {"name": "labelFontSizeMajor", "value": "20"},
+    {"name": "labelFontSizeMinor", "value": "14"},
+    {"name": "legendFontSize", "value": "12"},
+    {"name": "labelDataSrcFontSize", "value": "12"},
+    {"name": "basePeriodS", "value": "1951"},
+    {"name": "basePeriodE", "value": "1980"},
+    {
+      "name": "selectedStripeTemp",
+      "description": "Temperature in the selected year (stripe). To dispaly as a tick on the legend.",
+      "value": "0",
+      "on": [
+        {"events": "@stripe:mouseover", "update": "datum.temp"},
+        {"events": "@stripe:mouseout", "update": "0"}
+      ]
+    }
+  ],
+  "data": [
+    {
+      "name": "dataset",
+      "url": "https://raw.githubusercontent.com/avatorl/Deneb-Vega/main/warming-stripe/global-temp.csv",
```

Comment:
Should this be moved to vega datasets?

Reply:
Seems to be an old file. It's `"url": "data/global-temp.csv",` in current version of Vega specification for this example (https://github.com/vega/vega/blob/main/docs/examples/warming-stripes.vg.json)